java/code/src/com/redhat/rhn/domain/server/Server.java

```diff
@@ -2180,11 +2189,14 @@ public void updateServerInfo() {
             srvInfo.setServer(this);
             setServerInfo(srvInfo);
             srvInfo.setCheckinCounter(0L);
+            srvInfo.setUptimeData("");
         }
 
         srvInfo.setCheckin(new Date());
         srvInfo.setCheckinCounter(
                 Optional.ofNullable(srvInfo.getCheckinCounter()).orElse(0L) + 1);
+        srvInfo.setUptimeData(
+                String.valueOf(Optional.ofNullable(srvInfo.getUptimeData())));
```

Comment:
this is currently not really doing anything i would remove it for now until we find a place where to set the updated data.

Reply:
I would like to leave for now, I am totally open to and understand if this needs to be somewhere else, but removing it would mean removing its input and the test cases. I would much rather do that when I know how things should be done.